### PR TITLE
perf: use `neo-async` instead `async`

### PR DESCRIPTION
**Pull request by evilebottnawi**
### `Notable Changes`

- Increase performance

Reference: https://github.com/webpack/webpack/pull/6470

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,7 +47,8 @@
     "standard-version": "^4.2.0",
     "style-loader": "^0.18.2",
     "webpack-dev-server": "^2.4.1",
-    "webpack-merge": "^4.0.0"
+    "webpack-merge": "^4.0.0",
+    "webpack": "^3.11.0"
```
Comment (jhnns) [inline package.json]: Why is there a dev dependency on `webpack`?

Comment (evilebottnawi) [reply package.json]: @jhnns need for local tests, remove from PR?

Comment (michael-ciniawsky) [reply package.json]: @evilebottnawi Can you remove it for now please ? :)

Comment (evilebottnawi) [reply package.json]: @michael-ciniawsky let's do it :+1:

**Comment by jhnns** [comment]
@michael-ciniawsky @evilebottnawi thx 👍

**Pull request merged**